image_provisioner/playbooks/roles/jenkins/templates/variables.yaml.j2

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
@mbruzek can you update the inventory with the latest changes made to the image provisioner. https://github.com/openshift/svt/blob/master/image_provisioner/inventory/hosts.AWS can be used as a reference.